src/react/Router.jsx

```diff
@@ -70,7 +70,7 @@ export default ({ reactHistory: history, astarteClient, sessionManager, config,
           path="login"
           element={
             <Login
-              canSwitchLoginType={config.auth.length > 1}
+              authOptions={config.auth}
               defaultLoginType={config.default_auth || 'token'}
```

Comment:
I would expect `config.default_auth` to always be defined here, since we parsed it as required